servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Completable.java

```diff
@@ -1187,6 +1187,15 @@ public static Completable completed() {
         return CompletedCompletable.INSTANCE;
     }
 
+    /**
+     * Creates a realized completed {@code Completable} which first runs the supplied {@code Runnable}.
+     *
+     * @return A new {@code Completable}.
+     */
+    public static Completable fromRunnable(final Runnable runnable) {
+        return new RunnableCompletable(runnable);
```

Comment:
missing `Objects.requireNotNull(...)` check

Reply:
Wouldn't this be redundant with the check in `RunnableCompletable` itself?

Reply:
Yes sorry...